src/main/java/com/company/models/TwitterUser.java

```diff
@@ -0,0 +1,65 @@
+package com.company.models;
+
+import java.util.Objects;
+
+public class TwitterUser {
```

Comment:
why the change from User -> TwitterUser?

Reply:
I was getting myself confused between the Twitter4j User and my com.company.models User, so I just changed the models User to TwitterUser